MyTasks/Task2_1/UserAndBotDialogue.cs

```diff
@@ -0,0 +1,60 @@
+﻿
+Console.WriteLine("Здравствуйте! Введите Ваше имя:");
+var userName = Console.ReadLine();
+
+while (char.IsLower(userName, 0))
+{
+    Console.WriteLine("Пожалуйста, введите имя с заглавной буквы:");    // "обработка" неправильного пользовательского ввода
+    userName = Console.ReadLine();
+}
+
+foreach (var symbol in userName)   // где-то в этом блоке баг: если первый ввод некорректный, то идет
+                                    // бесконечный вывод "Имя должно состоять из букв" в ответ
+                                    // как на последующий некорректный ввод, так и на корректный.
+{
+    while (char.IsNumber(symbol) || char.IsPunctuation(symbol) || char.IsWhiteSpace(symbol))
+    {
+        Console.WriteLine("Имя должно состоять из букв.");
+        userName = Console.ReadLine();
+    }
+}
+
+Console.WriteLine("Введите Вашу фамилию:");
+var userSurname = Console.ReadLine();
+
+while (char.IsLower(userSurname, 0))
+{
+    Console.WriteLine("Пожалуйста, введите фамилию с заглавной буквы:");    // "обработка" неправильного пользовательского ввода
+    userSurname = Console.ReadLine();
+}
```

Comment:
It does not fit the task. Need a transformation, not a validation. Read the task
Don't force the user to type a million times his name

Reply:
Done.